client/src/components/ElementEditor/HoverBar.js

```diff
@@ -117,52 +117,55 @@ class HoverBar extends Component {
    * @returns {DOMElement}
    */
   render() {
-    const { AddElementPopoverComponent, elementTypes, baseAddHref, elementId } = this.props;
+    const {
+      AddElementPopoverComponent,
+      elementTypes,
+      elementId,
+      elementalAreaId
+    } = this.props;
     const { popoverOpen } = this.state;
 
-   return (
-     <div className="element-editor__add-block-area-container" id={`AddBlockArea_${elementId}`}>
-       <div
-         className="element-editor__add-block-area"
-         onMouseEnter={this.handleMouseEnter}
-         onMouseLeave={this.handleMouseLeave}
-       >
-
-         {/* render the hover bar in the corresponding style */}
-         {
-            (this.state.delayAreaActive && this.renderHoverBar(false)) ||
-            (this.state.instAreaActive && this.renderHoverBar(true))
-          }
-
-         {/* render the popover */}
-         {(this.state.delayAreaActive || this.state.instAreaActive) &&
-         <AddElementPopoverComponent
-           placement="bottom-end"
-           target={`AddBlockHoverBar_${elementId}`}
-           isOpen={popoverOpen}
-           elementTypes={elementTypes}
-           baseAddHref={baseAddHref}
-           toggle={this.toggle}
-           container={`#AddBlockArea_${elementId}`}
-         />
-          }
-       </div>
-     </div>
+    return (
+      <div className="element-editor__add-block-area-container" id={`AddBlockArea_${elementId}`}>
+        <div
+          className="element-editor__add-block-area"
+          onMouseEnter={this.handleMouseEnter}
+          onMouseLeave={this.handleMouseLeave}
+        >
+          {/* render the hover bar in the corresponding style */}
+          {(this.state.delayAreaActive && this.renderHoverBar(false)) ||
+            (this.state.instAreaActive && this.renderHoverBar(true))}
```

Comment:
I think I preferred it the way it was =/

Reply:
Hoverbar.js seems to be the one with all the auto-format crazyness - I'll tidy up the noise :)